GOT-StudentVersion/GOT-StudentVersion/viewControllers/DetailsViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  DetailsViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by Tia Lendor on 8/19/19.
+//  Copyright © 2019 C4Q . All rights reserved.
+//
+
+import UIKit
+
+class DetailsViewController: UIViewController {
+    
+    var episodeDetails: GOTEpisode!
+
+//    MARK -- Outlets and Labels
+    
+    @IBOutlet weak var detailedImageView: UIImageView!
+    
+    @IBOutlet weak var seasonLabel: UILabel!
+    
+    @IBOutlet weak var episodeLabel: UILabel!
+    
+    @IBOutlet weak var runtimeLabel: UILabel!
+    
+    @IBOutlet weak var airdateLabel: UILabel!
+    
+    @IBOutlet weak var detailedTextView: UITextView!
+    
+    @IBOutlet weak var episodeNumberLabel: UILabel!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        detailedImageView.image = UIImage(named: episodeDetails.originalImageID)
```

Comment:
this is good. Its best practice to wrap all of this in a function and then call it in the viewDidLoad